packages/vscode-dependencies-validation/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "dist",
+    "baseUrl": ".",
+    "skipLibCheck": true
```

Comment:
why     "skipLibCheck": true ?

Reply:
We have it in app-studio-toolkit.
Removed.